src/SFA.DAS.Assessors.Api/Models/GetStandardDetailsResponse.cs

```diff
@@ -0,0 +1,166 @@
+﻿using SFA.DAS.Assessors.InnerApi.Responses;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.Assessors.Api.Models
+{
+    public class GetStandardDetailsResponse
+    {
+        public string StandardUId { get; set; }
+        public string IfateReferenceNumber { get; set; }
+        public int LarsCode { get; set; }
+        public string Status { get; set; }
+        public string Title { get; set; }
+        public int Level { get; set; }
+        public decimal Version { get; set; }
+        public string OverviewOfRole { get; set; }
+        public string Keywords { get; set; }
+        public string Route { get; set; }
+        public string AssessmentPlanUrl { get; set; }
+        public string TrailBlazerContact { get; set; }
+        public string TypicalJobTitles { get; set; }
+        public string CoreSkillsCount { get; set; }
+        public List<string> Skills { get; set; }
+        public List<string> Knowledge { get; set; }
+        public List<string> Behaviours { get; set; }
+        public string StandardPageUrl { get; set; }
+        public string IntegratedDegree { get; set; }
+        public decimal SectorSubjectAreaTier2 { get; set; }
+        public string SectorSubjectAreaTier2Description { get; set; }
+
+        public List<ApprenticeshipFundingResponse> ApprenticeshipFunding { get; set; }
+
+        public StandardDatesResponse StandardDates { get; set; }
+
+        public StandardVersionDetailResponse VersionDetail { get; set; }
+
+        public EqaProviderResponse EqaProvider { get; set; }
+
+        public bool OtherBodyApprovalRequired { get; set; }
+        public string ApprovalBody { get; set; }
+        public List<string> Duties { get; set; }
+        public bool CoreAndOptions { get; set; }
+        public string CoreDuties { get; set; }
+        public bool IntegratedApprenticeship { get; set; }
+        public List<string> Options { get; set; }
+        public int MaxFunding { get; set; }
```

Comment:
Do we need to update these fields due to the changes to add "Proposed" ?

Reply:
yes... will have to wait till that is merged... I am too far from those changes :-(